ROADMAP.md

```diff
@@ -0,0 +1,41 @@
+# `sget` Roadmap
+
+### Today
+
+`sget` fetches blobs from OCI registries, using Docker API registry auth, checking content digests to ensure integrity, and verifying any `cosign` signatures that are found in the registry.
+
+`sget` currently has no functionality to fetch arbitrary URLs.
+
+### Phase 0
+
+Add basic URL-fetching functionality to `sget`, with an optional ability to check content digests to ensure integrity.
+
+This phase may also include a manual verification step where buffered contents are shown to the user, so they can manually inspect the contents before proceeding.
+
+### Phase 1: "Social Proof"
+
+In the absence of a maintainer-provided policy, `sget` users can gain assurance about the safety of an artifact by relying on crowd-sourced verificatioin and signatures.
+
+Signatures will be stored in Rekor, using certificates created by Fulcio's "keyless" OIDC capabilities.
+
+Before trusting an artifact, users should be able to:
+
+- see the total number of unique identities that have signed indicating that the artifact is safe to fetch.
+- see a random small subset of those identities.
+- see any identities with email addresses that match the URL's domain.
+- locally configure trusted identities, and trust those identities more than arbitrary random identities.
```

Comment:
>trust those identities more than arbitrary random identities

What does this mean?

Reply:
This is basically being able to say "I trust `dan@foo.bar` and `zac@bar.baz` -- if they've signed off on it, just lemme have it", and bypassing the less reliable broader "social proof" of a bunch of random folks on the internet.

If there's a better phrasing for this, let me know and I'll use that. "Random identities" isn't very descriptive, I agree.